data-structure/README.md

```diff
@@ -290,7 +309,7 @@ An example of a `Work`, and its physical and digital `Manifestation`s:
     "@type": "CreativeWork",
     "@id": "<URI pointing to this object>",
     "name": "Lord of the Rings",
-    "author": "<URI pointing to a Person or Organization object>"
+    "creator": "<URI pointing to a Person or Organization object>"
```

Comment:
It turns out "creator" is an alias to "author," but I find "author" awkward since you don't "author" everything; "creator" is more generic and fits better when you talk in terms of music, video, etc.